addons/escoria-core/game/core-scripts/log/esc_logger.gd

```diff
@@ -71,12 +102,24 @@ func info(string: String, args = []):
 # * string: Text to log
 # * args: Additional information
 func warning(string: String, args = []):
-	if _get_log_level() >= LOG_WARNING:
+	if _get_log_level() >= LOG_WARNING and !crashed:
 		var argsstr = str(args) if !args.empty() else ""
-		printerr("(W)\t" + string + " \t" + argsstr)
+		_log("(W)\t" + string + " \t" + argsstr, true)
 		if ProjectSettings.get_setting("escoria/debug/terminate_on_warnings"):
-			print_stack()
+			_perform_stack_trace_log()
+			var message: String = ProjectSettings.get_setting(
+				"escoria/debug/crash_message"
+			) % [
+					log_file.get_path_absolute().get_base_dir().plus_file(
+						escoria.save_manager.crash_savegame_filename.get_file()
```

Comment:
You added the save game file but didn't produce a savegame here, right?

Reply:
You are right. Fixed!